integration_tests/tests/common/mod.rs

```diff
@@ -322,21 +322,15 @@ pub async fn submit_measurements_and_verify_aggregate(
             )
             .await;
         }
-        VdafInstance::Prio3Histogram { buckets } => {
-            let vdaf = Prio3::new_histogram(2, buckets).unwrap();
+        VdafInstance::Prio3Histogram { length } => {
+            let vdaf = Prio3::new_histogram(2, *length).unwrap();
 
-            let mut aggregate_result = vec![0; buckets.len() + 1];
-            aggregate_result.resize(buckets.len() + 1, 0);
+            let mut aggregate_result = vec![0; *length];
+            aggregate_result.resize(*length, 0);
```

Comment:
Not introduced by this PR: This line is a no-op, because the vector is already the correct length. It's likely leftover from past clippy jousting or conflict resolution.
```suggestion
```